frontend/src/constants/constants.js

```diff
@@ -0,0 +1,2 @@
+export const MAP_VIEW = "MAP_VIEW";
```

Comment:
Could you name the file more relevant to the content? having constants.js file inside a constants folder is a little confusing